--- sound/soc/bcm/Kconfig	2017-03-29 23:38:09.569023168 -0700
+++ sound/soc/bcm/Kconfig.new	2017-03-30 17:55:37.371307298 -0700
@@ -147,6 +147,13 @@
 	help
 	  Say Y or M if you want to add support for Allo Piano DAC.
 
+config SND_BCM2708_SOC_MONOME
+       tristate "Support for monome-snd"
+       depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
+       select SND_SOC_CS4270
+       help
+       Say Y or M if you want to add support for Monome soundcard
+
 config SND_PISOUND
 	tristate "Support for Blokas Labs pisound"
 	depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
